modules/lss-missionKeyword/missionKeyword.user.js

```diff
@@ -11,323 +16,726 @@
+		142: 'Natuurbrand',
+		143: 'Natuurbrand',
+		144: 'Natuurbrand',
+		145: 'Natuurbrand',
+		146: 'Brandende vogelverschrikker',
+		147: 'Brandend pompoenveld',
+		148: 'Overkookte pompoensoep',
+		149: 'Heks vast in boom',
+		150: 'Monster uitgebroken',
+		151: 'Gestolen pompoenen',
+		152: 'Frankenstein gespot',
+		153: 'Snoep diefstal',
+		154: 'Horrorclown gespot',
+		155: 'Gaslekkage',
+		156: 'Gaslekkage',
+		157: 'Brand in ziekenhuis',
+		158: 'Brand in ziekenhuis',
+		159: 'Brand in sporthal',
+		160: 'Brandende cadeautjes',
+		161: 'Sinterklaas van dak gegleden',
+		162: 'Pepernoot op hoofd',
+		163: 'Piet bestolen',
+		164: 'Kind geschrokken',
+		165: 'Piet klem in schoorsteen',
+		166: 'Demonstratie zwarte piet',
+		167: 'Schoenen dief',
+		168: 'Verstikking in pepernoot',
+		169: 'Brandende kerstboom',
+		170: 'Brandende kerstboom',
+		171: 'Verkeersongeval door gladheid',
+		172: 'Gevallen kerstman',
+		173: 'Ruzie op de kerstmarkt',
+		174: 'Diefstal op de kerstmarkt',
+		175: 'Brandende kerstmarktkraam',
+		176: 'Gevallen schaatser',
+		177: 'Gevallen persoon door gladheid',
+		178: 'Donkere dagen offensief',
+		179: 'Snijwond door gebroken kerstbal',
+		180: 'Bergen object uit water voor politie',
+		181: 'Verkeersongeval met beknelling',
+		182: 'Personenauto in sloot',
+		183: 'Paard in sloot',
+		184: 'Eend verstrikt in visdraad',
+		185: 'Verlaten kinderfiets langs waterkant',
+		186: 'Parkeergarage onder water',
+		187: 'Kelder onder water',
+		188: 'Gevaarlijk hangende tak',
+		189: 'Boom op dak',
+		190: 'Straat onder water',
+		191: 'Losgewaaide dakbedekking',
+		192: 'Parkeerterrein onder water',
+		193: 'Koe in sloot',
+		194: 'Koe in gierput',
+		195: 'Lekkende woonboot',
+		196: 'Brand in woonwagen',
+		197: 'Brandende vrachtwagen',
+		198: 'Brand in woonkamer',
+		199: 'Inbraak in woning',
+		200: 'Drugslab aangetroffen',
+		201: 'Vechtpartij in café',
+		202: 'Steekincident',
+		203: 'Verkeersongeval met gevaarlijke stoffen',
+		204: 'Verkeersongeval met gevaarlijke stoffen',
+		205: 'Persoon te water',
+		206: 'Persoon te water',
+		207: 'Personenauto te water',
+		208: 'Personenauto te water',
+		209: 'Aanvaring met vrachtschip en motorboot',
+		210: 'Aanvaring met vrachtschip en motorboot',
+		211: 'Aanvaring met vrachtschip en motorboot',
+		212: 'Omgeslagen zeilboot',
+		213: 'Omgeslagen zeilboot',
+		214: 'Bosbrand',
+		215: 'Bosbrand',
+		216: 'Bosbrand',
+		217: 'Plofkraak',
+		218: 'Plofkraak',
+		219: 'Open dag, klein',
+		220: 'Open dag, groot',
+		221: 'Inval in woning',
+		222: 'Inbraak in woning',
+		223: 'Brand in kelder',
+		224: 'Brand in kelder',
+		225: 'Benauwdheid',
+		226: 'Val van trap',
+		227: 'Assistentie treinconducteur',
+		228: 'Assistentie treinconducteur',
+		229: 'Brand in tankstation'
+	};
+		
+	
+	var managedSettings = {
+		"id": LSS_MISSIONKEYWORD_STORAGE,
+		"title": I18n.t('lssm.missionKeywordDefaults.name'),
+		"version": VERSION,
+		"settings": {}
+	};
+	
+	var tmpObject = {
```

Comment:
Wenn du auf die Funktion bestehst, solange sie noch nicht Out-of-the-Box im ManagedSettings Modul umgesetzt ist, kannst du das so machen.
Aber warum setzt du das Objekt nicht direkt in die Settings?